src/textual/widgets/_list_view.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
+
+import asyncio
+from asyncio import Future
+
+from textual import events
+from textual.binding import Binding
+from textual.containers import Vertical
+from textual.geometry import clamp
+from textual.message import Message
+from textual.reactive import reactive
+from textual.widgets._list_item import ListItem
+
+
+class ListView(Vertical, can_focus=True, can_focus_children=False):
+    DEFAULT_CSS = """
+    ListView {
+        scrollbar-size-vertical: 1;
+    }
+    """
+
+    BINDINGS = [
+        Binding("down", "down", "Down"),
+        Binding("up", "up", "Up"),
+        Binding("enter", "select", "Select"),
+    ]
+
+    index = reactive(0, always_update=True)
+
+    def __init__(
+        self,
+        *children: ListItem,
+        initial_index: int | None = 0,
+        name: str | None = None,
+        id: str | None = None,
+        classes: str | None = None,
+    ) -> None:
+        super().__init__(*children, name=name, id=id, classes=classes)
+        self.index = initial_index
+
+    @property
+    def highlighted_child(self) -> ListItem | None:
+        """Get the currently highlighted ListItem
```

Comment:
I think we're going to adopt property style docstrings in this format:

"""ListItem | None: description"""